### ISSUE TITLE
When user starts bot, let them know whether their partner has joined

### ISSUE THREAD

**Issue by nicktohzyu** [Tembusu-Student-Developer-Club/AngelMortalTelegraf#6]
If partner has already joined, message the partner when user joins

**Comment by nicktohzyu** [comment]
Say we have some players A, B, C, within others. A is the angel of B, B is the angel of C (there may be other relationships). Let's assume that A has already registered (so B and C have not). When B registers, they should see something like "Your angel has already registered. You may begin chatting with them" (because his angel A has already registered). And B should also see "Your mortal has not registered yet" (because his mortal C has not registered)

After this, we will work on a separate feature to notify B when his partner has registered.

Before you get to coding, propose how you would check whether the player's partners have already registered

**Comment by aanghs22** [comment]
- Create a new export function(?) under module.exports within the messages.js file to have a message that states "Your (target) has not registered yet".
- Within the implementation of RegisterSuccessHandler, add an else clause to both 
`    if (angel.isRegistered()) {
        await ctx.model.mortalBot.telegram.sendMessage(angel.telegramId, messages.RegisteredNotifier('mortal'))
    }

    if (mortal.isRegistered()) {
        await ctx.model.angelBot.telegram.sendMessage(mortal.telegramId, messages.RegisteredNotifier('angel'))
    }`  to send the above message if the angel/mortal is not registered.

Does this work?

**Comment by nicktohzyu** [comment]
Yup looks good. Perhaps can add the `else` clauses to say that the partner has not registered yet! 

For future planning discussion no need to give code haha. Something like "use angel.isRegistered()" will do (cos I don't remember the whole code base). When you're writing code, might as well use the PR feature to propose it! Then I can give line by line comments.

Remember to merge master into your branch (if you've created the branch before the new updates).
